x-pack/plugins/maps/public/connected_components/right_side_controls/layer_control/layer_toc/toc_entry/toc_entry_actions_popover/toc_entry_actions_popover.tsx

```diff
@@ -97,15 +153,41 @@ export class TOCEntryActionsPopover extends Component<Props, State> {
     ];
 
     if (!this.props.isReadOnly) {
+      if (this.state.supportsFeatureEditing) {
+        actionItems.push({
+          name: EDIT_FEATURES_LABEL,
+          icon: <EuiIcon type="vector" size="m" />,
+          'data-test-subj': 'editLayerButton',
+          toolTipContent: this.state.editModeEnabled
```

Comment:
Why is the tooltip content dependent on editModeEnabled? Would it make sense to always show tooltip content?

Reply:
The tooltip is telling users why edit mode is disabled, similar to what we did [here](http://github.com/aaronjcaldwell/kibana/blob/a9311dd83ae8f8b8df0583eab12ec18f7390bd90/x-pack/plugins/maps/public/connected_components/right_side_controls/layer_control/layer_toc/toc_entry/toc_entry_actions_popover/toc_entry_actions_popover.tsx#L132-L132).

Reply:
ok, I see that now. How about renaming "editModeEnabled" to "canEditFeatures"? I was confused by "editModeEnabled" since its very similar to drawModeActive. I miss took the "Enabled" to mean edit mode was turned on.